tools/quickstart/quickstart/cli.py

```diff
@@ -1,52 +1,146 @@
+import functools
 from textwrap import fill
 
 import click
+import pkg_resources
 
 from quickstart import bridge, docker, monitor, netstats, validator_account
-from quickstart.constants import NETSTATS_SERVER_LAIKA_BASE_URL
+
+DEFAULT_CONFIGS = ["Laika"]
+LAIKA, = DEFAULT_CONFIGS
+
+LAIKA_DOCKER_COMPOSE_FILE = "laika-docker-compose.yaml"
+LAIKA_BRIDGE_CONFIG_FILE = "laika-bridge-config.toml"
+LAIKA_NETSTATS_SERVER_BASE_URL = "https://laikanetstats.trustlines.foundation/"
+
+
+def config_file_getter(filename):
+    return functools.partial(
+        pkg_resources.resource_filename, __name__, f"configs/{filename}"
+    )
+
+
+class DefaultPathType(click.Path):
+    def __init__(self, default_path_lookups, *args, **kwargs):
+        """
+        Like the click path type, but additionally accepts a mapping for predefined paths
+        :param default_path_lookups: mapping from template name to lookup function
+        :param args: args for click.Path
+        :param kwargs: kwargs for click.Path
+        """
+        super().__init__(*args, **kwargs)
+
+        self.default_path_lookups = default_path_lookups
+
+    def convert(self, value, param, ctx):
+        if value in self.default_path_lookups:
+            value = self.default_path_lookups[value]()
+
+        return super().convert(value, param, ctx)
 
 
 @click.command()
+@click.option(
+    "--config",
+    "-c",
+    help="Config to use. [Default: Laika]",
```

Comment:
This help is pretty short and not useful. Which of all the configs? It turns out that it defines presets for all following options. Maybe this should be mentioned here. Else I would think: If I set Laika here, I go for setting it at the other options as well. But then I get the error message that I can not combine them. Thereby the usage is not obvious.

Reply:
After all, I'm thinking if a command group is not more clear to use. So we have one, where you can define a preset and one where everything must be manually set. This would also make the top body of this function more understandable. Moreover the whole default path and getter can be removed. This would also allow to simply add function descriptions which are then parsed as help text by click. The user clearly see that he can either go the "automatic" way or the manual one.
Share you opinion and feel free to simply close this.

Reply:
what would the command group be called?

Reply:
Actually a group is not necessary. Both commands do not share their root functionality. So it could be simply to different commands. The only point for a group would the shared option `--project-name`. Not sure if it is worth to establish a group for that, since it requires to maintain the context object. We would end up with something ugly like `quickstart --project-name trustlines4ever manual --bridge-config ...`.

So a proposal with pretty literal command names would be something like:
`quickstart use-defaults <chain-name>` ~> have a switch-case for the `chain-name` argument
`quickstart ask-for-settings <options>` ~> all options are simply required and must not be checked

Reply:
The command names are pretty bad, so with this names I would leave it like we have it. 
Also remember the default is that you just do `quickstart`, so that also has to work.

Reply:
we could have laika as a command 
so `quickstart` that can take options
or `quickstart laika` that does not take options
`quickstart` without options will fall back to quickstart laika

Reply:
That looks actually neat. And then later add `quickstart trustnet` (`tlbc`...) ?

Reply:
yep

Reply:
done